components/TextControls/src/BaseTextAreas/private/MDCBaseTextAreaLayout.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+#import "MDCTextControl.h"
+#import "MDCTextControlAssistiveLabelView.h"
+
+@interface MDCBaseTextAreaLayout : NSObject
+
+@property(nonatomic, assign) CGRect labelFrameFloating;
```

Comment:
we spoke offline briefly about making all these readonly. is that still happening?

Reply:
I can do that, yep.

Reply:
Done.